breathecode/events/views.py

```diff
@@ -105,12 +106,26 @@ def get_events(request):
     return Response(serializer.data)
 
 
+@permission_classes([AllowAny])
```

Comment:
This view includes a post, a unauthenticated user should not be able to add new events